optaweb-employee-rostering-frontend/cypress.json

```diff
@@ -1,5 +1,5 @@
 {
-    "baseUrl": "http://localhost:3000",
+    "baseUrl": "http://localhost:8180",
```

Comment:
Does this port work also if you run the frontend via npm?

Reply:
Checking

Reply:
Yes, I forgot about that. Configured for node
[see](https://github.com/kiegroup/optaweb-employee-rostering/pull/506/commits/a810f7848c0c56642f7fe9a44eaa01adfe019855 )